src/components/connection-prompt/bluetooth/BluetoothConnectDialog.svelte

```diff
@@ -36,12 +50,24 @@
     {$t('connectMB.pattern.heading')}
   </DialogHeading>
   <p>{$t('connectMB.pattern.subtitle')}</p>
-  <div class="flex justify-center p-20">
-    <PatternMatrix matrix={$patternMatrixState} onMatrixChange={updateMatrix} />
+  <div class="pt-20 pb-10">
+    <div class="flex justify-center">
+      <PatternMatrix
+        matrix={$patternMatrixState}
+        onMatrixChange={updateMatrix}
+        aria-invalid={attemptedToPairWithInvalidPattern} />
```

Comment:
Almost, but we only want aria-invalid on inputs where the value is 0 / not entered rather than all of them.

Reply:
https://github.com/microbit-foundation/ml-trainer/pull/175/commits/2affd69c1d677c5d77a91ca273a4ff41dbebc131